packages/Search/src/DTK_DistributedSearchTree_decl.hpp

```diff
@@ -96,10 +98,11 @@ class DistributedSearchTree
            Kokkos::View<double *, DeviceType> &distances ) const;
 
   private:
+    friend Details::DistributedSearchTreeImpl<DeviceType>;
     Teuchos::RCP<Teuchos::Comm<int> const> _comm;
-    BVH<DeviceType> _local_tree;
-    BVH<DeviceType> _distributed_tree;
-    SizeType _size;
+    BVH<DeviceType> _top_tree;    // replicated
+    BVH<DeviceType> _bottom_tree; // local
```

Comment:
If I didn't know what `bottom_tree` and `top_tree` were, there is no way I would understand what they are.

Reply:
Did we put anywhere what the trees are? I am just worry that if someone is new on the project there is no way to know that these are